index.js

```diff
@@ -66,12 +65,18 @@ app.use('/', auth, cors(), morgan('combined'), graphqlHTTP(request => {
   const accessToken = request.headers['x-access-token'];
   const userID = request.headers['x-user-id'];
 
+  let timezone;
+  if (request.headers.timezone && request.headers.timezone.split(';')[2].length) {
```

Comment:
Seeing as this is a non-standards header, it should be prefixed by `x-`, like the others above.